dtale/views.py

```diff
@@ -969,7 +964,7 @@ def view_main(data_id=None):
     :type data_id: str
     :return: HTML
     """
-    if data_id is None or data_id not in global_state.get_data().keys():
+    if data_id is None or int(data_id) not in global_state.keys():
```

Comment:
Once again, can save some code by implementing a `global_state.contains` function that can handle string/int/None data ids